iroha/src/sumeragi.rs

```diff
@@ -17,8 +17,12 @@ use rand::{rngs::StdRng, seq::SliceRandom, SeedableRng};
 use std::{
     fmt::{self, Debug, Formatter},
     sync::Arc,
+    time::{Duration, SystemTime},
 };
 
+/// Specifies the amount of time in which the peer expects commit message from proxy after voting on the block.
```

Comment:
Be simple and formal - "Amount of time Peer waits for `CommitMessage` from the proxy tail."